### ISSUE TITLE
Issue creating AWS profile

### ISSUE THREAD

**Issue by cjh1** [Kitware/HPCCloud#410]
I think this is a problem introduced by the move from the paraviewweb repo?

```
HPCCloud.js:46976 Uncaught TypeError: _network2.default.saveAWSProfile is not a function(anonymous function) @ HPCCloud.js:46976(anonymous function) @ HPCCloud.js:46984dispatch @ HPCCloud.js:44006onUpdateItem @ HPCCloud.js:58313saveItem @ HPCCloud.js:58234formAction @ HPCCloud.js:58237onAction @ HPCCloud.js:48258ReactErrorUtils.invokeGuardedCallback @ HPCCloud.js:6642executeDispatch @ HPCCloud.js:6442executeDispatchesInOrder @ HPCCloud.js:6465executeDispatchesAndRelease @ HPCCloud.js:5895executeDispatchesAndReleaseTopLevel @ HPCCloud.js:5906forEachAccumulated @ HPCCloud.js:6743processEventQueue @ HPCCloud.js:6111runEventQueueInBatch @ HPCCloud.js:6772handleTopLevel @ HPCCloud.js:6788handleTopLevelWithoutPath @ HPCCloud.js:17019handleTopLevelImpl @ HPCCloud.js:16999perform @ HPCCloud.js:8944batchedUpdates @ HPCCloud.js:12990batchedUpdates @ HPCCloud.js:8449dispatchEvent @ HPCCloud.js:17130
```

**Comment by cjh1** [comment]
@TristanWright Can we try and resolve this one?

**Comment by cjh1** [comment]
Fixes by #411
